src/android/io/sqlc/SQLitePlugin.java

```diff
@@ -409,11 +427,13 @@ public void run() {
                                 dbq.cbc.error("couldn't delete database");
                             }
                         } catch (Exception e) {
+                            logger.logError("Error deleteDatabaseNow: " + e.getMessage(), "SQLite");
```

Comment:
We should improve this error message.

Reply:
How? Make it more readable? e.g.: "Error while trying to deleting the ciphered database: " + e.getMessage()?

Reply:
Yes, something like. If these messages are supposed to be displayed in Service Center, they should be more user-friendly and not simple debug messages. Also, do not forget to send the stack trace too 😄

Reply:
Fixed ✅ What you think? I also add the logs [here](https://docs.google.com/spreadsheets/d/1ePJXRMfQuOvHC34LcctEZA6q61OUvrjr9lwCqI6BtPk/edit#gid=139767747)

Reply:
The error messages seem right to me. You just forget to add the stack-trace 😄

Reply:
Good catch!! Thx